arcgis-ios-sdk-samples/Layers/Browse WFS layers/WFSLayersTableViewController.swift

```diff
@@ -75,6 +75,7 @@ class WFSLayersTableViewController: UITableViewController {
             let cell = tableView.dequeueReusableCell(withIdentifier: "LayerCell", for: indexPath)
             let layerInfo = allLayerInfos[indexPath.row]
             cell.textLabel?.text = layerInfo.title
+            cell.accessoryType = (tableView.indexPathForSelectedRow == indexPath) ? .checkmark : .none
```

Comment:
Since you're already setting the accessory type of the selected cell in `didSelectRowAt`, I would suggest setting the accessory type of the unselected cell in `didDeselectRowAt` (which gets called automatically when another row becomes selected, when multiple selection is disallowed).

Reply:
Yes I do call `didDeselectRowAt`, that deselects it but `cellForRowAt:` returns `nil` if the cell is invisible. So I couldn't set the `accessoryType` to `.none` in `didDeselectRowAt`. It works for visible cells as expected

Reply:
Right. I missed that. Thanks!